tests/installation/product_selection/install_SLES.pm

```diff
@@ -5,11 +5,15 @@
 # Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
 
 use base 'y2_installbase';
+use testapi 'get_var';
 use strict;
 use warnings;
 
 sub run {
-    $testapi::distri->get_product_selection()->install_product('SLES');
+    my $timeout = 120 * get_var('TIMEOUT_SCALE', 1);
```

Comment:
isn't the default for libyui-rest-api 30 seconds?

Reply:
It's 120 seconds, I've seen in the log. and you can check from [**yui-log**](https://openqa.suse.de/tests/10609779/logfile?filename=yui-log.txt) it start from `[2023-03-02T10:22:27.772353+01:00] `to to

Reply:
unfortunately we did that :(
please create a method `get_timeout` in `lib/YuiRestClient.pm` to match both values and not hardcode it.